pica/ml/classifiers/xgbm.py

```diff
@@ -32,9 +32,9 @@ def __init__(self, max_depth: int = 4, learning_rate: float = 0.05,
             'n_estimators':[20, 30, 50, 80, 100, 150, 200, 300, 500],
             'subsample': np.arange(0.2, 1., 0.1).round(2),
             'colsample_bytree': np.arange(0.2, 1., 0.1).round(2),
-            'min_child_weight': np.arange(1, 20),
+            'min_child_weight': np.arange(1, 20).astype(int),
             'gamma': [0, 0.2, 0.5, 1, 5, 10],
```

Comment:
Here we have a mixture of Python lists and Numpy arrays. This does not matter as long as we only ever iterate over these structures. I'm only slightly worried that we at some point may do `gamma * 2` and obtain something rather surprising.